### YamlMime:JavaType
uid: "com.azure.resourcemanager.hdinsight.models.Errors"
fullName: "com.azure.resourcemanager.hdinsight.models.Errors"
name: "Errors"
nameWithType: "Errors"
summary: "The error message associated with the cluster creation."
inheritances:
- "<xref href=\"java.lang.Object\" data-throw-if-not-resolved=\"False\" />"
inheritedMembers:
- "java.lang.Object.clone()"
- "java.lang.Object.equals(java.lang.Object)"
- "java.lang.Object.finalize()"
- "java.lang.Object.getClass()"
- "java.lang.Object.hashCode()"
- "java.lang.Object.notify()"
- "java.lang.Object.notifyAll()"
- "java.lang.Object.toString()"
- "java.lang.Object.wait()"
- "java.lang.Object.wait(long)"
- "java.lang.Object.wait(long,int)"
syntax: "public final class Errors"
constructors:
- "com.azure.resourcemanager.hdinsight.models.Errors.Errors()"
methods:
- "com.azure.resourcemanager.hdinsight.models.Errors.code()"
- "com.azure.resourcemanager.hdinsight.models.Errors.message()"
- "com.azure.resourcemanager.hdinsight.models.Errors.validate()"
- "com.azure.resourcemanager.hdinsight.models.Errors.withCode(java.lang.String)"
- "com.azure.resourcemanager.hdinsight.models.Errors.withMessage(java.lang.String)"
type: "class"
metadata: {}
package: "com.azure.resourcemanager.hdinsight.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-hdinsight:1.0.0-beta.1
